README.md

```diff
@@ -23,9 +23,19 @@ f = Firefox.fetch()
 
 his = f.get()
 ```
-
  - `Firefox` in the above snippet can be replaced with any of the [supported browsers](https://browser-history.readthedocs.io/en/latest/browsers.html).
- - `his` is a list of `(datetime.datetime, url)` tuples.
+ - `his` is a list of `(datetime.datetime, url)` tuples from the firefox browser.
+
+
+```python
+from browser_history.utils import get_history
+
+outputs = get_history()
+
+histories = outputs.get()
+```
+
+ - ``histories`` is a list of ``(datetime.datetime, url)`` tuples from every browser installed on the system.
```

Comment:
This can be structured better. I suggested something lke:
```
To get history from all installed browsers, ....

If you want history from a specific browser, ...
```

Also, `get_history` can be imported directly from `browser_history` since it is imported in `__init__.py`.